journalbeat/plan.sh

```diff
@@ -0,0 +1,35 @@
+pkg_name=journalbeat
+pkg_origin=core
+pkg_version="6.6.0"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("Apache-2.0")
+pkg_deps=(core/glibc core/systemd)
+pkg_build_deps=(core/go core/git core/make core/gcc core/systemd core/patchelf)
+pkg_bin_dirs=(bin)
+pkg_svc_user=root
+pkg_svc_group=root
+pkg_description="Journalbeat is a lightweight journald log shipper for Elasticsearch."
+pkg_upstream_url="https://github.com/elastic/beats/tree/master/journalbeat"
+
+do_download() {
+  return 0
+}
+
+do_unpack() {
+  return 0
+}
+
+do_build() {
+  SYSTEMD_LIB_PATH=$(pkg_path_for core/systemd)
```

Comment:
I would recommend using `SYSTEMD_INCLUDE_PATH=$(pkg_path_for core/systemd)/include` since this is only used in the `CGO_FLAGS` lines below.

Reply:
Swapped the var name and moved /include up from the two CSGO_FLAGS lines.